config/runtime.exs

```diff
@@ -69,3 +69,11 @@ scheduler_jobs =
 config :screenplay, Screenplay.Scheduler, jobs: scheduler_jobs
 
 config :screenplay, Screenplay.Repo, pool_size: 10
+
+if config_env() in [:prod, :test] do
```

Comment:
**suggestion:** we should be able to support local auth for `:test` also. probably just needs a `config, :screenplay, Screenplay.Repo, database: "screenplay_test"` in `config/test.exs` or something similar.

generating a new, empty phoenix project might be a helpful reference on how ecto is configured across environments by default

Reply:
Oof, thank you. Shouldn't have brute forced the solution into this file.